src/main/java/racingcar/controller/RacingGameController.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.controller;
+
+import camp.nextstep.edu.missionutils.Console;
+import racingcar.service.RacingService;
+import racingcar.service.IOService;
+import racingcar.model.RandomMoveStrategy;
+
+import java.util.List;
+
+public class RacingGameController {
+
+    private final IOService ioService;
+
+    public RacingGameController() {
+        this.ioService = new IOService();
```

Comment:
[우아한테크코스 Java Style](https://github.com/JunHoPark93/google-java-styleguide) 해당 문서 5.3 캐멀케이스 부분을 보시면
```suggestion
        this.ioService = new IoService();
```
이와 같은 컨벤션을 지키기를 권장합니다!

Reply:
넵 ! 감사합니다